src/resources/spanProcessor.ts

```diff
@@ -0,0 +1,32 @@
+import { BatchSpanProcessor, ReadableSpan } from '@opentelemetry/sdk-trace-base';
+import { logger } from '../logging';
+
+export class LumigoSpanProcessor extends BatchSpanProcessor {
+  override onEnd(span: ReadableSpan) {
+    if (shouldSkipSpanExport(span)) {
+      logger.debug('Not exporting span because it has NO_EXPORT=true attribute');
+      return;
+    }
+
+    super.onEnd(span);
+  }
+}
+
+/**
+ * Given a readable span, returns true if the span should be skipped from export
+ * @param span A readable span to check
+ */
+export const shouldSkipSpanExport = (span: ReadableSpan): boolean => {
+  return span.attributes && span.attributes.SKIP_EXPORT === true;
```

Comment:
check what the conventions are, i would have expected something like `span.attributes["skip-export"]`

Reply:
Keeping the convention the same as in the otel python distro, with uppercase SKIP_EXPORT